### ISSUE TITLE
Learning rate modifictions in abcnn

### ISSUE THREAD

**Issue by virajshastri97** [meshde/EruditeX#42]
Train and test with the exponential decay function of tensorflow for learning rate in abcnn
